VirtualH89/Src/NilMemory8K.h

```diff
@@ -0,0 +1,29 @@
+///
+/// \file NilMemory8K.h
+///
+/// Implementation of an 8K page of non-existent memory. Reads 0, writes ignored.
+///
+/// \date Mar 05, 2016
+/// \author Douglas Miller
+///
+#ifndef NILMEMORY8K_H_
+#define NILMEMORY8K_H_
+
+#include "Memory8K.h"
+
+class NilMemory8K: public Memory8K
```

Comment:
We could keep it like this for now. When/If we add support for other systems like the DG Super-89 that has a different page size.